### ISSUE TITLE
qwen2-vl-7b  lora sft; grad_norm: nan; why? learning rate 1e-5; and donot find the max_grad_norm

### ISSUE THREAD

**Issue by weilanzhikong** [2U1/Qwen2-VL-Finetune#10]
0%|          | 1/666 [00:22<4:08:53, 22.46s/it]
                                                 
{'loss': 1.0446, 'grad_norm': nan, 'learning_rate': 1.4925373134328357e-08, 'epoch': 0.0}

  0%|          | 1/666 [00:24<4:08:53, 22.46s/it]
  0%|          | 2/666 [00:40<3:41:45, 20.04s/it]
                                                 
{'loss': 0.0, 'grad_norm': nan, 'learning_rate': 2.9850746268656714e-08, 'epoch': 0.01}

**Comment by 2U1** [comment]
I'm not sure with only the information you provided.
It could be a dataset problem. or Some other hyper paremter problem.

**Comment by Wang-Xiaodong1899** [comment]
Hi @weilanzhikong , I also encountered the same issue. Any suggestion to solve this issue?

**Comment by Wang-Xiaodong1899** [comment]
Zero-2 has this error: grad_norm: Nan, but Zero-3 is OK.

**Comment by easonyang1996** [comment]
> Zero-2 has this error: grad_norm: Nan, but Zero-3 is OK.

try `overlap_comm=False` in zero2.json
https://github.com/deepspeedai/DeepSpeed/issues/5347
